app/assets/javascripts/discourse/tests/integration/components/admin-page-header-test.gjs

```diff
@@ -174,4 +187,60 @@ module("Integration | Component | AdminPageHeader", function (hooks) {
     await click(".edit-groupings-btn");
     assert.true(actionCalled);
   });
+
+  test("@headerActionComponent is rendered with actions arg", async function (assert) {
+    await render(<template>
+      <AdminPageHeader
+        @headerActionComponent={{AdminPageHeaderActionsTestComponent}}
+      />
+    </template>);
+
+    assert
+      .dom(".admin-page-header-actions-test-component .award-badge")
+      .exists();
+  });
+});
+
+module("Integration | Component | AdminPageHeader | Mobile", function (hooks) {
+  hooks.beforeEach(() => {
+    forceMobile();
+  });
```

Comment:
it doesn't matter in this case but it's usually better to use non-arrow function in before/afterEach, so that `this` is correct (we might eventually add an auto-lint for that 😅)

Reply:
Okay cool will change